blocks/library/image/editor.scss

```diff
@@ -4,7 +4,7 @@
 
 	img {
 		display: block;
-		width: 100%;
+		max-width: 100%;
 	}
```

Comment:
I think you can also remove the max-with, we already have:
```
.gutenberg__editor img {
	max-width: 100%;
}
```

Reply:
@Soean You're right :) 👍